crates/compiler/builtins/roc/Str.roc

```diff
@@ -504,7 +504,7 @@ walkUtf8WithIndexHelp = \string, state, step, index, length ->
     else
         state
 
-## Make sure at least some number of bytes fit in this string without reallocating
+## Enlarge the Str for at least capacity additional bytes
```

Comment:
This argument is unnamed, but the comment refers to it as if it's named "capacity".
I know it could only be one thing but it still feels wrong for the docs to refer to a name that isn't defined.
That's probably why the previous comment referred only to the type.

Reply:
hmm. That's a really good point. This matches `List.reserve` so I am going to leave it for now and file an issue on the general question.